chrisparsons83/FFDiscordBot
# Fix/162 adp update

**Testing**  

1. checkout
2. run bot with a valid bot key
3. run adp command
4. ensure bot has replaced MFL with 4f4
5. ensure bot responds with numbers corresponding to the source - https://www.4for4.com/adp

https://github.com/chrisparsons83/FFDiscordBot/issues/162

https://github.com/chrisparsons83/FFDiscordBot/compare/master...DiaryRoom:fix/162_adp_update
